attributes/default.rb

```diff
@@ -88,9 +92,16 @@
 # Set prefix to '' if you want Chef tags to be sent without prefix.
 default['datadog']['tag_prefix'] = 'tag:'
 
-# Don't change these
-# The host of the Datadog intake server to send agent data to
-default['datadog']['url'] = 'https://app.datadoghq.com'
+# Don't change these.
+# The host of the Datadog intake server to send Agent data to, only set this option
+# if you need the Agent to send data to a custom URL.
+# For Agent 6, defaults to nil because "url" overrides the site setting defined in "site".
+default['datadog']['url'] =
+  if node['datadog']['agent6'] == true
+    nil
```

Comment:
I think `url` is also used to configure the endpoint of the handler (see https://github.com/DataDog/chef-datadog/blob/v2.16.1/recipes/dd-handler.rb#L71 and https://github.com/DataDog/chef-datadog/blob/v2.16.1/recipes/dd-handler.rb#L30), will this still work fine?